docs/docs-content/clusters/edge/edge-configuration/installer-reference.md

```diff
@@ -113,21 +113,25 @@ The `stylus.site` blocks accept the following parameters.
 | --- | --- |
 | `paletteEndpoint` | The URL endpoint that points to Palette. Example: `api.spectrocloud.com` | 
 | `edgeHostToken` | A token created at the tenant scope that is required for auto registration. |
-| `projectUid` | The id of the project the Edge host will belong to. |
+| `projectUid` | The ID of the project the Edge host will belong to. |
 | `projectName` | The name of the project. 
-| `name` | If you do not specify an Edge hostname, the system will generate one from the serial number of the device. If Stylus is unable to identify the serial number, it will generate a random ID instead. In cases where the hardware does not have a serial number, we suggest that you specify a value so there is minimal chance of duplication. Use the value `"$random"` to generate a random ID. You can also use the `DeviceUIDPaths` to read in a value from a system file.  |
```

Comment:
@lennessyy the last two sentences inform the user of two additional behaviors, random, and DeviceUIDPaths. Are these two features no longer available?

Reply:
i moved this down into an info box because this is getting way too long a block of text and no longer readable. I also cannot seem to have two paragraphs in a single table cell.